Makefile

```diff
@@ -25,20 +40,38 @@ serve: clients
 clients-force:
 	npm run build-clients
 
-clients:
-ifeq ("$(wildcard dist/api-client)","")
+clients: dist/api-client dist/import-gitlab-client dist/import-github-client dist/import-azuredevops-client
+
+dist/api-client:
 	npx ng build api-client
-endif
-ifeq ("$(wildcard dist/import-gitlab-client)","")
+
+dist/import-gitlab-client:
 	npx ng build import-gitlab-client
-endif
-ifeq ("$(wildcard dist/import-github-client)","")
+
+dist/import-github-client:
 	npx ng build import-github-client
-endif
-ifeq ("$(wildcard dist/import-azuredevops-client)","")
+
+dist/import-azuredevops-client:
 	npx ng build import-azuredevops-client
-endif
-	@# This comment silences warning "make: Nothing to be done for 'clients'."
 
-deps:
-	npm install
+lint: lint-ng lint-md lint-scss
+
+lint-fix: lint-ng-fix lint-md-fix lint-scss-fix
+
+lint-ng:
+	ng lint
+
+lint-ng-fix:
+	ng lint --fix
```

Comment:
This is probably a good consideration for all makefiles- using ng directly without npx encurages global installation of ng-cli such that its on the path. This would likely cause version diffs between the intended package gotten through `npm install` for a given project and the opposing globally isntalled ng-cli.
Using npx to run ng howver bypasses this problem. As npm/npx is expected to be on the path regardless of project this is easier to use. It will also default to using the ng-cli installed to the project based on the local project.

Reply:
You're right! Nice catch! I blindly copied the scripts from `package.json`, but in `package.json` it selects the locally installed one for us.

Will fix

Reply:
I've updated. It's using `npx` everywhere else in the other repos for the `remark lint` that is just a few lines above this `lint-ng-fix`.